editors/kakoune/Portfile

```diff
@@ -0,0 +1,25 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                kakoune
+version             2020.09.01
+github.setup        mawww kakoune ${version} v
+categories          editors
+platforms           darwin
+license             Unlicense
+maintainers         {@felix userspace.com.au:felix} openmaintainer
+
+description         Modal editor, fewer keystrokes, multiple selections, Orthogonal design
+long_description \
+    Kakoune is a code editor that implements Vi’s "keystrokes as a text editing \
+    language" model. As it’s also a modal editor, it is somewhat similar to the \
+    Vim editor (after which Kakoune was originally inspired).
+
+homepage            http://kakoune.org/
+
+checksums           rmd160  793126df59847c8663be08688b32ded3f47c9314 \
+                    sha256  106adf110d1a4c17853b4783a4f4240270ca2b515f2cce5e04345d76767d97d9 \
+                    size    612988
+
+use_configure       no
```

Comment:
Because this isn't using a configure script where it could be done for you automatically, you're responsible for arranging for the right compiler to be used along with the right -arch flags. The makefile 1.0 portgroup can help you do so; read its source code to find out how.

Reply:
This hasn't been changed yet, which is why setting `compiler.cxx_standard 2017` isn't having an effect yet.